src/prommis/precipitate/tests/test_precipitator.py

```diff
@@ -247,9 +252,18 @@ def test_conservation(self, prec):
         )
 
         reversed_react = dict(map(reversed, prec.fs.properties_solid.react.items()))
+        pass_through_elements = ["Ca", "H", "Cl", "SO4", "H2O", "HSO4"]
         for j in prec.fs.properties_aq.dissolved_elements:
-            if j == "Ca":
-                pass
+            if j in pass_through_elements:
```

Comment:
Minor comment, but in this case it would be easier to write `value(a) == pytest.approx(value(b), rel=1e-5, abs=1e-5)`